### PR TITLE
set git jobs number when checkout submodule

### PR DESCRIPTION
**Pull request by frank-dong-ms**
Use environment variable to set git jobs number when checkout submodule, reason for that is I see git stuck when checkout submodule with docker.

### CODE REVIEW

File: manywheel/build_common.sh [hunk 1]
```diff
@@ -100,7 +100,14 @@ if [[ ! -d "$pytorch_rootdir" ]]; then
 else
     pushd $pytorch_rootdir
 fi
-git submodule update --init --recursive --jobs 0
+
+if [[ -n "$GIT_JOBS_NUMBER" ]]; then
```
Comment (driazati) [inline manywheel/build_common.sh:104]: 1. can we rename to `GIT_UPDATE_NUM_JOBS` for clarity?
2. bash has a language feature for this pattern, could you switch to that to make this simpler? https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash

Comment (frank-dong-ms) [reply manywheel/build_common.sh:104]: Sure, will rename it and use the recommend pattern, thanks for pointing out.

**Review by driazati — APPROVED** [review]

**Comment by driazati** [comment]
Code looks fine, might need a rebase though before we can merge

**Comment by frank-dong-ms** [comment]
I looked at main fork, looks like submodule checkout has been moved out to circle ci, we will no longer need this PR, I will close this PR, thanks @driazati for review.

**Pull request closed without merging**